scripts/smoke_tests/setup.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+import sys
+import json
+import time
+import threading
+
+import requests
+
+from tests.functional.backend.common import BaseFunctionalTestCase
+
+# min amount of published collections expected by smoke tests
+MIN_TEST_COLLECTIONS = 4
```

Comment:
Question: why have a MIN_TEST_COLLECTIONS instead of always have 10?

Reply:
Yeah, I think I'll change this. Initially, I was thinking about this distinction as a time-saving procedure--as in, if we already have >= the minimum amount needed to pass the test, it might not be worthwhile to add to the deployment time to get to the Max every time (vs. with less than the min, we already have to take the time to trigger at least 1 create/upload/publish workflow, so we may as well kick off enough in parallel to reach the max). But if we're already asserting confidence that 10 will significantly reduce flakiness over 4, it probably saves time long-term to make sure we always have at least 10.